origin.spec

```diff
@@ -0,0 +1,138 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global gopath      %{_datadir}/gocode
+%global import_path github.com/openshift/origin
+%global commit      d3f40eafae8ae7bbca61981e33f384375307fafa
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           origin
+Version:        0.2
+#Release:        1git%{shortcommit}%{?dist}
+Release:        2%{?dist}
+Summary:        Open Source Platform as a Service by Red Hat
+License:        ASL 2.0
+URL:            https://%{import_path}
+ExclusiveArch:  x86_64
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+
+# FIXME - Need to add a -devel subpackage to etcd that provides the golang
+#         libraries/packages, but this will work for now.
+BuildRequires:  systemd
+BuildRequires:  golang >= 1.2-7
+
+Requires:       docker-io
```

Comment:
It looks like the RHEL7 packaging has provided docker-io as far back as 1.2.0-1.1 which shipped back in September. We should probably specify a minimum version though, from what I understand we require 1.3.2 right now.